ramls/defaultRules/defaultRules.json

```diff
@@ -0,0 +1,101 @@
+{
+  "rules": [
+    {
+      "ruleId": "5105b55a-b9a3-4f76-9402-a5243ea63c95",
+      "name": "password_length",
+      "type": "RegExp",
+      "validationType": "Strong",
+      "state": "Enabled",
+      "moduleName": "mod-password-validator",
+      "expression": "^.{8,}$",
+      "description": "The password length must be minimum 8 digits",
+      "orderNo": 0,
+      "errMessageId": "password.length.invalid"
+    },
+    {
+      "ruleId": "dc653de8-f0df-48ab-9630-13aacfe8e8f4",
+      "name": "alphabetical_letters",
+      "type": "RegExp",
+      "validationType": "Strong",
+      "state": "Enabled",
+      "moduleName": "mod-password-validator",
+      "expression": "(?=.*[a-z])(?=.*[A-Z]).+",
+      "description": "The password must contain both upper and lower case letters",
+      "orderNo": 1,
+      "errMessageId": "password.alphabetical.invalid"
+    },
+    {
+      "ruleId": "3e3c53ae-73c2-4eba-9f09-f2c9a892c7a2",
+      "name": "numeric_symbol",
+      "type": "RegExp",
+      "validationType": "Strong",
+      "state": "Enabled",
+      "moduleName": "mod-password-validator",
+      "expression": "(?=.*\\d).+",
+      "description": "The password must contain at least one numeric character",
+      "orderNo": 2,
+      "errMessageId": "password.number.invalid"
+    },
+    {
+      "ruleId": "2e82f890-49e8-46fc-923d-644f33dc5c3f",
+      "name": "special_character",
+      "type": "RegExp",
+      "validationType": "Strong",
+      "state": "Enabled",
+      "moduleName": "mod-password-validator",
+      "expression": "(?=.*[!\"#$%&'()*+,-./:;<=>?@\\[\\]^_`{|}~]).+",
+      "description": "The password must contain at least one special character",
+      "orderNo": 3,
+      "errMessageId": "password.no-special-character"
+    },
+    {
+      "ruleId": "2f390fa6-a2f8-4027-abaf-ee61952668bc",
+      "name": "no_user_name",
+      "type": "RegExp",
+      "validationType": "Strong",
+      "state": "Enabled",
+      "moduleName": "mod-password-validator",
+      "expression": "^(?:(?!<USER_NAME>).)+$",
+      "description": "The password must contain at least one special character",
+      "orderNo": 4,
+      "errMessageId": "password.no-special-character"
+    },
+    {
+      "ruleId": "8d4a2124-8a54-4c49-84c8-36a8f7fc01a8",
+      "name": "sequence_characters",
+      "type": "RegExp",
+      "validationType": "Strong",
+      "state": "Enabled",
+      "moduleName": "mod-password-validator",
+      "expression": "^(?:(?!qwe)(?!asd)(?!zxc)(?!qaz)(?!zaq)(?!xsw)(?!wsx)(?!edc)(?!cde)(?!rfv)(?!vfr)(?!tgb)(?!bgt)(?!yhn)(?!nhy)(?!ujm)(?!mju)(?!ik,)(?!,ki)(?!ol.)(?!.lo)(?!p;/)(?!/;p)(?!123).)+$",
+      "description": "The password must contain at least one special character",
+      "orderNo": 5,
+      "errMessageId": "password.no-special-character"
```

Comment:
Could we use camelCase notation for `errMessageId` as it is the common pattern for translation keys on UI.